CONTRIBUTING.md

```diff
@@ -225,6 +225,36 @@ discussed extensively on the mailing list and IRC, be accompanied by a widely
 discussed BIP and have a generally widely perceived technical consensus of being
 a worthwhile change based on the judgement of the maintainers.
 
+### Finding Reviewers
+
+As most reviewers are themselves developers with their own projects, the review
+process can be quite lengthy, and some amount of patience is required. If you find
+that you've been waiting for a pull request to be given attention for several
+months, there may be a number of reasons for this, some of which you can do something
+about:
+
+  - It may be because of a feature freeze due to an upcoming release. During this time,
+    only bug reports are taken into consideration. If your pull request is a new feature,
+    it will be disregarded until the release is over. Wait for release.
```

Comment:
I think 'disregarded' is too strong here. It will be delayed, but there's no rule preventing e.g. discussion or people from testing it.